apps/server/src/services/PlaybackService.js

```diff
@@ -127,6 +127,29 @@ export class PlaybackService {
     }
   }
 
+  /**
+   * Starts playback on next event
+   * @param {string} [fallbackAction] - 'stop', 'pause', or null
+   */
+  static startNext(fallbackAction) {
+    const nextEvent = eventLoader.findNext();
+    if (nextEvent) {
+      const success = PlaybackService.loadEvent(nextEvent);
+      if (success) {
+        socketProvider.info('PLAYBACK', `Loaded event with ID ${nextEvent.id}`);
+        PlaybackService.start();
+      }
+    } else if (fallbackAction === 'stop') {
+      socketProvider.info('PLAYBACK', `No next event found! Stopping playback`);
+      PlaybackService.stop();
+    } else if (fallbackAction === 'pause') {
+      socketProvider.info('PLAYBACK', `No next event found! Pausing playback`);
+      PlaybackService.pause('pause');
+    } else {
+      socketProvider.info('PLAYBACK', `No next event found! Continuing playback`);
+    }
```

Comment:
nice, I like how you left the default behaviour as a fallback. very elegant

Reply:
thanks